src/tools/fuzzing.h

```diff
@@ -815,6 +815,7 @@ class TranslateToFuzzReader {
       case f32:
```

Comment:
I think the fuzzing support is just not present yet, in that `getType` doesn't return an except_ref? (that is, no except_refs will be generated by the fuzzer) probably worth adding a clear TODO on that near the top somewhere (as otherwise, the preparations for except_ref support scattered throughout might imply that it does work)

Reply:
Done